assets/js/main.js

```diff
@@ -0,0 +1,5 @@
+document.addEventListener("click", function (e) {
+  if (e.target && e.target.dataset.dismissable !== undefined) {
+    document.querySelector(`#${e.target.dataset.dismissable}`).innerHTML = "";
```

Comment:
does this need to be dynamic - couldnt the modal just always be called modal, even if multiple different things are rendered it in? then you just need the same prop everywhere to dismiss it

Reply:
probably, yes. Because you're never going to have more than 1 open at a time